switchStatement.js

```diff
@@ -36,3 +53,28 @@ Example:
   - Recommendations: Wear sunscreen, a hat, and sunglasses.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+weatherForecast = prompt('What\'s the Weather Forecast Today?');
+
+switch (weatherForecast) {
+  case "sunny":
+  console.log('Recommendations: Wear a waterproof jacket, boots, and bring an umbrella.')
+  break;
+  case "cloudy":
+  console.log('Recommendations: Wear sunscreen, a hat, and sunglasses.')
+  break;
+  default:
```

Comment:
You can't add the default keyword in the middle of cases because it won't go through other cases that are below it

Reply:
that's right  ... I fix it  now.